src/cephadm/cephadm.py

```diff
@@ -2486,8 +2486,11 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
         if ctx.shared_ceph_folder:  # make easy manager modules/ceph-volume development
             ceph_folder = pathify(ctx.shared_ceph_folder)
             if os.path.exists(ceph_folder):
+                cephadm_binary = ceph_folder + '/src/cephadm/cephadm'
+                if not os.path.exists(pathify(cephadm_binary)):
+                    raise Error("cephadm binary does not exist. Please run './build.sh cephadm' from ceph/src/cephadm/ directory.")
+                mounts[cephadm_binary] = '/usr/sbin/cephadm'
```

Comment:
FYI, Redouane's PR got merged so you should be able to rebase and remove this commit